### PR TITLE
chore: improve error style

### PR DESCRIPTION
**Pull request by Ananya2001-an**
## What kind of change does this PR introduce?

Using lipgloss style to format error message. Created `utils.Red` for errors....implemented in only a few places for now.

### CODE REVIEW

**Comment by sweatybridge** [comment]
Before you dive too deep into this, you can change the styling of all error messages in [one place](https://github.com/supabase/cli/blob/main/cmd/root.go#L96).

```go
func Execute() {
	if err := rootCmd.Execute(); err != nil {
		fmt.Fprintln(os.Stderr, utils.Red(err.Error()))
...
```

To prevent printing duplicated errors, you can also set `SilenceErrors: true` for rootCmd.

**Comment by Ananya2001-an** [comment]
Oh that would be nice... since I have never worked on a Go cli before I was not aware of this! Thanks for the heads up :)

File: internal/utils/colors.go [hunk 1]
```diff
@@ -13,6 +15,11 @@ func Yellow(str string) string {
 	return lipgloss.NewStyle().Foreground(lipgloss.Color("11")).Render(str)
 }
 
+// For errors.
+func Red(str string) error {
+	return errors.New(lipgloss.NewStyle().Foreground(lipgloss.Color("9")).Render(str))
```
Comment (sweatybridge) [inline internal/utils/colors.go:20]:
```suggestion
func Red(str string) string {
	return lipgloss.NewStyle().Foreground(lipgloss.Color("9")).Render(str)
```

File: internal/utils/colors.go [hunk 2]
```diff
@@ -1,6 +1,8 @@
 package utils
 
 import (
+	"errors"
+
```
Comment (sweatybridge) [inline internal/utils/colors.go:5]:
```suggestion
```

**Comment by Ananya2001-an** [comment]
Sure that sounds good!

**Review by sweatybridge — APPROVED** [review]

**Pull request merged**